### PR TITLE
Add brand new logo to the project

### PR DESCRIPTION
**Pull request by caneco**
Following your request in the tweet:
https://twitter.com/Iamvarandas/status/1179471663152402439

Here it is your brand new logo. Hope you like it!

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,4 +209,10 @@ export interface ILocationRequestParams {
 
 ---
 
+## Credits
+
+Logo by [Caneco](httsp://www.twitter.com/caneco).
```
Comment (valterc) [inline README.md]: Typo: httsp should be https 😉

Comment (caneco) [reply README.md]: Well catched. Thanks 😄

Comment (AndreVarandas) [reply README.md]: Hi @caneco do you want to submit another MR to fix this?

**Pull request merged**